src/dxbc/dxbc_compiler.h

```diff
@@ -79,12 +79,14 @@ namespace dxvk {
   struct DxbcRegisterPointer {
     DxbcVectorType    type;
     uint32_t          id;
+    uint32_t          boundsCheck;
```

Comment:
I'm really not a big fan of adding a member that's only ever going to be used by one thing.